script.js

```diff
@@ -0,0 +1,53 @@
+var nums = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15].sort(function() {
+  return .5 - Math.random();
+});
```

Comment:
We talked about this a bit in-person, but how could you avoid a) hard-coding all of these numbers, and b) making the number of values correspond to the `x` and `y` specified below?